yellowbrick/yb_palettes.py

```diff
@@ -86,40 +86,58 @@ def as_hex(self):
 
 def color_palette(palette=None, n_colors=None, desat=None):
     """Return a list of colors defining a color palette.
-    Availible seaborn palette names:
-        accent, dark, paired, pastel, bold, muted
-    Availible seaborn palette names:
-        sns_deep, sns_muted, sns_bright, sns_pastel, sns_dark, sns_colorblind
-    Other options:
-        list of colors
     Calling this function with ``palette=None`` will return the current
     matplotlib color cycle.
     This function can also be used in a ``with`` statement to temporarily
     set the color cycle for a plot or set of plots.
-    Parameters
-    ----------
-    palette: None, string, or sequence, optional
+
+    :param palette:
         Name of palette or None to return current palette. If a sequence, input
         colors are used but possibly cycled and desaturated.
-    n_colors : int, optional
+
+        Available seaborn palette names from :py:mod:`seaborn.palettes` are:
+
+        .. hlist::
+            :columns: 3
+
+            * :py:const:`deep`
+            * :py:const:`dark`
+            * :py:const:`paired`
+            * :py:const:`pastel`
+            * :py:const:`bold`
+            * :py:const:`muted`
+            * :py:const:`sns_deep`
+            * :py:const:`sns_muted`
+            * :py:const:`sns_bright`
+            * :py:const:`sns_pastel`
+            * :py:const:`sns_dark`
+            * :py:const:`sns_colorblind`
+
+    :type palette: None or str or sequence
+    :param n_colors:
```

Comment:
These info blocks e.g. `:param:`, `:type`:, etc would be used in each docstring. This way sphinx knows how to render them based on whatever HTML theme you use. I put in the read the docs theme but there are actually several, and you can create your own if you wanna get fancy.